src/SFA.DAS.PensionsRegulatorApi.UnitTests/Application/Queries/GetValidatedOrganisations/Given_Repository_Returns_Results.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture;
+using FluentAssertions;
+using MediatR;
+using NSubstitute;
+using NUnit.Framework;
+using PensionsRegulatorApi.Application.Queries;
+using PensionsRegulatorApi.Data;
+using PensionsRegulatorApi.Domain;
+
+namespace SFA.DAS.PensionsRegulatorApi.UnitTests.Application.Queries.GetValidatedOrganisations
+{
+    [TestFixture]
+    [ExcludeFromCodeCoverage]
+    public class Given_Repository_Returns_Results
+    {
+        private IOrganisationRepository _mockRepository;
+        private IRequestHandler<global::PensionsRegulatorApi.Application.Queries.GetValidatedOrganisations, IEnumerable<Organisation>> _sut;
+        private IEnumerable<Organisation> _repositoryResults;
+
+        public Given_Repository_Returns_Results()
+        {
+            _repositoryResults =
+                new Fixture()
+                    .CreateMany<Organisation>(
+                        new Random()
+                            .Next(1, 15));
+        }
+
+        [SetUp]
+        public void Given()
+        {
+            _mockRepository = Substitute.For<IOrganisationRepository>();
+
+            _mockRepository
+                .GetOrganisationsForPAYEReferenceAndAORN(Arg.Any<string>(), Arg.Any<string>())
+                .Returns(_repositoryResults);
+
+            _sut = new GetValidatedOrganisationsHandler(_mockRepository);
+        }
+
+        [Test]
+        public async Task Then_Equivalent_Results_Are_Returned()
+        {
+            var organisations =
+                await
+                    _sut
+                        .Handle(
+                            new global::PensionsRegulatorApi.Application.Queries.GetValidatedOrganisations(
+                                "anynonemptytext",
+                                "anynonemptytext"),
```

Comment:
can just be "" or string.Empty I think given your setup

Reply:
Following some of @MichaelYoung1981 's comments these query tests have been completely re-written.